EventView-1/src/main/java/com/eventview/repo/EventRepoImpl.java

```diff
@@ -37,11 +38,23 @@
 
     @Override
     public Events findByEventsId(Integer eventId) {
-        String SELECT_EVENT_BYID = "select * from events where event_id = ?";
+        String SELECT_EVENT_BY_ID = "SELECT * FROM events WHERE event_id =?";
         try {
-            Events events = jdbcTemplate.queryForObject(SELECT_EVENT_BYID, new Object[]{eventId}, new EventRowMapper());
-            log.info("query generated " + SELECT_EVENT_BYID + "-----" + eventId);
+            Events events = jdbcTemplate.queryForObject(SELECT_EVENT_BY_ID, new Object[]{eventId}, new EventRowMapper());
+            log.info("query generated " + SELECT_EVENT_BY_ID + "-----" + eventId);
             return events;
+        } catch (Exception e) {
+            throw new EventTypeNotFoundException("Event not founf with id: " + eventId);
+        }
+    }
+
+    @Override
+    public EventsPayload findByEventsIdCustom(Integer eventId) {
+        String SELECT_EVENT_BY_ID_CUS = "select e.event_id,u.first_name,u.last_name, et.event_type,e.event_date from events e left join users u on e.user_id = u.user_id left join eventtypes et on et.event_type_id=e.event_type_id where event_id = ?";
```

Comment:
explain why u need this

Reply:
this returns the full name and all by id which is consistent with the getAllEvents